UseAsBuilder fails without details

```
> Configure project :
Project : => 'org.jabref' Java module

> Task :rewriteRun
Validating active recipes
Parsing sources from project JabRef
All sources parsed, running active recipes: org.jabref.config.rewrite.cleanup
line 1:0 mismatched input '<EOF>' expecting {'.', '!', '*', '..', Identifier}
The recipe produced an error. Please report this to the recipe author.

> Task :rewriteRun FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':rewriteRun'.
> java.lang.NullPointerException: Cannot invoke "String.isEmpty()" because "this.pattern" is null
```

Codebase: https://github.com/JabRef/jabref/

--- a/src/main/java/org/openrewrite/staticanalysis/UseAsBuilder.java
+++ b/src/main/java/org/openrewrite/staticanalysis/UseAsBuilder.java
@@ -33,7 +33,10 @@ import static org.openrewrite.java.format.TabsAndIndents.formatTabsAndIndents;
 @Value
 @EqualsAndHashCode(callSuper = false)
 public class UseAsBuilder extends Recipe {
-    @Option
+    @Option(
+            displayName = "Builder Type",
+            description = "Fully qualified name of the Builder"
+    )
     String builderType;
 
     @Option(

--- a/src/main/java/org/openrewrite/staticanalysis/UseAsBuilder.java
+++ b/src/main/java/org/openrewrite/staticanalysis/UseAsBuilder.java
@@ -27,6 +27,7 @@ import org.openrewrite.java.tree.*;
 
 import java.util.*;
 
+import static org.openrewrite.Validated.notBlank;
 import static org.openrewrite.java.format.TabsAndIndents.formatTabsAndIndents;
 
 @Value
@@ -60,6 +61,11 @@ public class UseAsBuilder extends Recipe {
     @Override
     public String getDescription() {
         return "When an API has been designed as a builder, use it that way rather than as a series of setter calls.";
+    }
+
+    @Override
+    public Validated validate() {
+        return super.validate().and(notBlank("builderType", builderType));
     }
 
     @Override

